Tasks/MavenAuthenticateV0/task.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "D4B964F9-EA90-41BB-9526-29589628AD90",
+    "name": "MavenAuthenticate",
+    "friendlyName": "Maven Authenticate",
+    "description": "Provides credentials for Azure Artifacts feeds and service connections for maven repositories",
+    "helpUrl": "https://go.microsoft.com/fwlink/?linkid=2099968",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=2099968)",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
```

Comment:
To be consistent with other tasks, please change the minor version to sprint number.

Reply:
Fixed